samples/AzureSample.ResourceManager.Sample/src/Generated/Models/AdditionalCapabilities.Serialization.cs

```diff
@@ -125,6 +130,10 @@ private BinaryData SerializeBicep(ModelReaderWriterOptions options)
             return BinaryData.FromString(builder.ToString());
         }
 
+        private void TransformFlattenedOverrides(BicepModelReaderWriterOptions bicepOptions, IDictionary<string, string> propertyOverrides)
```

Comment:
can we omit if there aren't any instead of adding this dead code?

Reply:
When emitting the methods we won't know if it is needed or not.